README.md

```diff
@@ -0,0 +1,61 @@
+# Mosaic 
+
+> :warning: **This package in under early development and not ready for production use, yet.**
+
+`mosaic` provides the functionality to visualize unstructured mesh data on it's native grid within `matplotlib`. 
+Currently `mosaic` only supports MPAS meshes, but future work will add support for other unstructured meshes used in `E3SM`.
+
+## (Developer) Installation 
+
+Assuming you have a working `conda` installation, you can install the latest development version of `mosaic` by running: 
+```
+conda env create --file dev-environment.yml
+conda activate mosaic-dev
+
+python -m pip install -e .
+```
+
+If you have an existing `conda` environment you'd like install the development version of `mosaic` in, you can run: 
+```
+conda install --file dev-environment.yml
+
+python -m pip install -e .
+```
+
+## Example Usage
+
+First we need to download a valid MPAS mesh. To do so run:
+```
+curl https://web.lcrc.anl.gov/public/e3sm/inputdata/ocn/mpas-o/EC30to60E2r3/mpaso.EC30to60E2r3.230313.nc -o mpaso.EC30to60E2r3.230313.nc
+```
+
+Then we can use `mosaic` to plot on the native mesh using `matplotlib`. For example:
+```python
+
+import cartopy.crs as ccrs
+import mosaic
+import matplotlib.pyplot as plt
+import xarray as xr
+
+ds = xr.open_dataset("mpaso.EC30to60E2r3.230313.nc")
+
+projection = ccrs.InterruptedGoodeHomolosine()
+transform = ccrs.PlateCarree()
+
+fig, ax = plt.subplots(1, 1, figsize=(9,7), facecolor="w",
+                       constrained_layout=True,
+                       subplot_kw=dict(projection=projection))
+
+descriptor = mosaic.Descriptor(ds, projection, transform)
+
+patches = mosaic.polypcolor(ax, descriptor, ds.nCells, antialiaseds=False)
```

Comment:
I found this example confusing because `ds.nCells` is a dimension, not a field.  I'm surprised that this works. I changed this to `ds.indexToCellID` (which is an actual field) and the results look the same.  I guess I just didn't know that you could treat an xarray dimension this way and I think it doesn't make for a particularly intuitive example (at least for me).
```suggestion
patches = mosaic.polypcolor(ax, descriptor, ds.indexToCellID, antialiaseds=False)
```

Reply:
Totally fair. Happy to switch to `indexToCellID`. Thanks for catching this.